CONTRIBUTING.md

```diff
@@ -4,25 +4,43 @@ void-packages is the backbone of the Void Linux distribution. It contains all th
 
 This document describes how you, as a contributor, can help with adding packages, correcting bugs and adding features to void-packages.
 
-## Getting your packages into Void by yourself
+## Creating and updating packages in Void by yourself
 
-If you really want to get a package into Void Linux, we recommend you package it yourself.
+If you really want to get a new package or package update into Void Linux, we recommend you contribute it yourself.
 
 We provide a [comprehensive Manual](./Manual.md) on how to create new packages.
-There's also a [manual for xbps-src](./README.md), which is used
-to build package files from templates.
+There's also a [manual for xbps-src](./README.md), which is used to build package files from templates.
 
 For this guide, we assume you have basic knowledge about [git](http://git-scm.org), as well as a [GitHub Account](http://github.com).
 
 To get started, [fork](https://help.github.com/articles/fork-a-repo) the void-linux `void-packages` git repository on GitHub and clone it:
 
+    # with SSH set up for GitHub (recommended)
     $ git clone git@github.com:<user>/void-packages.git
+    # without SSH set up for GitHub
+    $ git clone https://github.com/<user>/void-packages.git
 
 To keep your forked repository up to date, setup the `upstream` remote to pull in new changes:
 
     $ git remote add upstream https://github.com/void-linux/void-packages.git
     $ git pull --rebase upstream master
 
+This can also be done with the `github-cli` tool:
+
+    $ gh repo fork void-linux/void-packages
+    $ gh repo clone <user>/void-packages
+
+This automatically sets up the `upstream` remote, so `git pull --rebase upstream master` can still be used to keep your fork up-to-date.
```

Comment:
Are you sure the remote is always called upstream after the clone? I think I remember some issues with remote naming, but I don't know what it was.

Reply:
yes

```
$ gh repo clone --help
...
If the repository is a fork, its parent repository will be added as an additional
git remote called "upstream". The remote name can be configured using `--upstream-remote-name`.
The `--upstream-remote-name` option supports an "@owner" value which will name
the remote after the owner of the parent repository.
...
```

Reply:
That sounds like it's called upstream only if it's a fork and we are cloning void-linux/void-packages which isn't a fork